bodhi/server/models.py

```diff
@@ -1506,6 +1506,21 @@ def get_testcase_karma(self, testcase):
                         bad += 1
         return bad * -1, good
 
+    def beautify_title(self, amp=False):
+        if len(self.builds) == 1:
+            return packagename_from_nvr(self, self.builds[0].nvr)
+        elif len(self.builds) == 2:
+            return " and ".join([packagename_from_nvr(self, build.nvr) for build in self.builds[:2]])
+        elif len(self.builds) > 2:
+            title = ", ".join([packagename_from_nvr(self, build.nvr) for build in self.builds[:2]])
+            if amp:
+                title += " &amp; "
+            else:
+                title += " and "
```

Comment:
For both of these, we could employ the [Oxford Comma](http://www.theonion.com/americanvoices/oxford-comma-wins-court-case-workers-55578?utm_medium=RSS&utm_campaign=feeds) by putting a comma at the beginning of the string, like ```", &amp; "``` and ```", and "```. This is at your option.